### PR TITLE
some cleanups from stubtest

### PR DESCRIPTION
**Pull request by twoertwein**

### CODE REVIEW

File: pandas-stubs/api/indexers/__init__.pyi [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from pandas.core.indexers import check_array_indexer as check_array_indexer
-from pandas.core.window.indexers import (
+from pandas.core.indexers import (
```
Comment (twoertwein) [inline pandas-stubs/api/indexers/__init__.pyi:1]: `indexers` is a file in the stubs but a folder in pandas. `BaseIndexer` and so on are technically important from a file in this folder not from `__init__`.

File: pandas-stubs/io/common.pyi [hunk 2]
```diff
@@ -17,15 +17,7 @@ def validate_header_arg(header) -> None: ...
...
 ) -> FilePathOrBuffer[AnyStr]: ...
-def is_s3_url(url) -> bool: ...
-def is_gcs_url(url) -> bool: ...
 def urlopen(*args, **kwargs) -> IO: ...
-def get_filepath_or_buffer(
-    filepath_or_buffer: FilePathOrBuffer,
-    encoding: str | None = ...,
-    compression: str | None = ...,
-    mode: str | None = ...,
-): ...
```
Comment (Dr-Irv) [inline pandas-stubs/io/common.pyi:28]: I have a feeling that this entire file can be deleted, because it has nothing that is public in it.

**Review by Dr-Irv — CHANGES_REQUESTED** [review]

File: pandas-stubs/io/common.pyi (hunk 2 above)
Comment (twoertwein) [reply pandas-stubs/io/common.pyi:28]: I agree. Happy to delete it - wanted to avoid "controversial" changes.

**Review by Dr-Irv — APPROVED** [review]
Thanks @twoertwein

**Pull request merged**